deploy/kube.yaml

```diff
@@ -50,52 +50,25 @@ spec:
         app: contacts-app
     spec:
       containers:
-      - name: gateway
-        args: 
-        - "-health"
-        - "0.0.0.0:8088"
-        - "-shealth"
-        - "0.0.0.0:8089"
-        image: infoblox/contacts-gateway:latest
-        imagePullPolicy: Always
-        ports:
-        - containerPort: 8080
-        livenessProbe:
-          httpGet:
-            path: /healthz
-            port: 8088
-            scheme: HTTP
-          initialDelaySeconds: 5
-          successThreshold: 1
-          timeoutSeconds: 10
-        readinessProbe:
-          httpGet:
-            path: /ready
-            port: 8088
-            scheme: HTTP
-          initialDelaySeconds: 5
-          periodSeconds: 5
-          successThreshold: 1
-          failureThreshold: 10
       - name: server
         args:
         - "-db"
         - "host=postgres port=5432 user=postgres password=postgres sslmode=disable dbname=contacts"
-        - "-health"
-        - "0.0.0.0:8089"
         # uncomment to add authorization to the contacts-app example. Please
         # note that authz also needs to be running in order to authorize
         # requests to the contacts-app
         # - "-authz=pdpserver.authz:5555"
-        image: infoblox/contacts-server:latest
-        imagePullPolicy: Always
+        image: infoblox/atlas-contacts-app-server:latest
+        imagePullPolicy: IfNotPresent
         ports:
-        - containerPort: 9091
+        - containerPort: 8080
+          protocol: TCP
+        - containerPort: 9090
           protocol: TCP
         livenessProbe:
           httpGet:
             path: /healthz
-            port: 8089
+            port: 8080
```

Comment:
does health and ready use the same port as gateway?

Reply:
Is this a good idea? For example current CSP exposes it's app root path to the outside world via ingress (e.g. exposes everything on a certain port) . In this case `/health` and `/ready` checks will become available from the outside, while ideally it should be available only inside K8s cluster.

Reply:
hmm, that's a really good point -- I didn't think of that. I'll work on that change, thanks!